rebuild/test/spec/specTestVersioning.ts

```diff
@@ -0,0 +1,14 @@
+import path from "path";
+
+// WARNING! Don't move or rename this file !!!
+//
+// This file is used to generate the cache ID for spec tests download in Github Actions CI
+// It's path is hardcoded in: `.github/workflows/test-spec.yml`
+//
+// The contents of this file MUST include the URL, version and target path, and nothing else.
+
+export const SPEC_TEST_REPO_URL = "https://github.com/ethereum/consensus-spec-tests";
+export const SPEC_TEST_VERSION = "v1.2.0";
```

Comment:
note that i updated to the most recent version

Reply:
latest version seems to be [v1.3.0](https://github.com/ethereum/consensus-spec-tests/releases/tag/v1.3.0)